### PR TITLE
Punjabi ('pa') Language added

### PR DESCRIPTION
**Pull request by markandey1414**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [x] Optimization
- [ ] Documentation Update

## Description
Added 1 new language, Punjabi ('pa') at the path `lib\l10n\arb\app_pa.arb` and updated `app.dart`

## Featured Covered in this PR

- [x] Task 1
- [ ] Task 2

## Related Tickets & Documents

- Related Issue #
- Closes #

## Screenshots, Recordings

*Please replace this line with instructions on how to test your changes, or any screenshots or recording you can attach here.*

## Tested Feature??

- [x] In Real Device.
- [ ] In Emulator

### CODE REVIEW

**Comment by markandey1414** [comment]
have a look now, I made another PR for punjabi

File: lib/l10n/arb/app_pa.arb [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+  "pageNotFound":"ਪੰਨਾ ਮਿਲਿਆ ਨਹੀਂ",
+  "settings":"ਵਿਵਸਥਾ",
+  "enterNewEmail":"ਨਵੀਂ ਈਮੇਲ ਦਾਖਲ ਕਰੋ",
+  "enterRegisteredEmail":"ਰਜਿਸਟਰਡ ਈਮੇਲ ਦਰਜ ਕਰੋ",
+  "enterCurrentPassword":"ਮੌਜੂਦਾ ਪਾਸਵਰਡ ਦਿਓ",
+  "resetPassword":"ਪਾਸਵਰਡ ਰੀਸੈੱਟ",
+  "autoSync":"ਆਟੋ ਸਿੰਕ",
+  "alreadyHaveAnAccount":"ਕੀ ਪਹਿਲਾਂ ਤੋਂ ਹੀ ਖਾਤਾ ਹੈ?",
+  "syncNow":"ਸਿੰਕ",
```
Comment (SankethBK) [inline lib/l10n/arb/app_pa.arb:16]: This appears to be wrong, its translating to `sink` instead. I got this on google translate `ਹੁਣੇ ਸਿੰਕ ਕਰੋ` can you check once

File: lib/l10n/arb/app_pa.arb [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  "appVersion":"ਐਪਲੀਕੇਸ਼ਨ ਵਰਜਨ",
+  "continueAsGues":"ਮਹਿਮਾਨ ਵਜੋਂ ਜਾਰੀ ਰੱਖੋ",
+  "byContinuingYouAgree":"ਜਾਰੀ ਰੱਖ ਕੇ, ਤੁਸੀਂ ਸਾਡੇ ਨਾਲ ਸਹਿਮਤ ਹੁੰਦੇ ਹੋ ",
+  "privacyPolicy":"ਪਰਾਈਵੇਟ ਨੀਤੀ",
+  "closeTheApp":"ਐਪ ਨੂੰ ਬੰਦ ਕਰਨਾ ਹੈ?",
+  "changePassword":"ਪਾਸਵਰਡ ਬਦਲੋ",
+  "changeEmail":"ਈਮੇਲ ਬਦਲੋ",
+  "setupYourAccount":"ਆਪਣਾ ਖਾਤਾ ਸੈਟ ਅਪ ਕਰੋ",
+  "logIn":"ਲਾਗਿਨ",
+  "logOut":"ਲਾੱਗ ਆਊਟ, ਬਾਹਰ ਆਉਣਾ",
```
Comment (SankethBK) [inline lib/l10n/arb/app_pa.arb:56]: I am getting `Log out, log out` on google translate, we only need it once

File: lib/l10n/arb/app_pa.arb (hunk 1 above)
Comment (markandey1414) [reply lib/l10n/arb/app_pa.arb]: yes, updated now

**Review by SankethBK — APPROVED** [review]
LGTM, thanks for contributing

**Pull request merged**